libs/sdk-utils/multicall/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "root": "libs/sdk-utils/multicall",
+  "sourceRoot": "libs/sdk-utils/multicall/src",
+  "projectType": "library",
+  "targets": {
+    "lint": {
```

Comment:
There should be a `build` target. Did you check "create buildable project" when you generated this one?

Reply:
Don't think so, I will add a build target for all repos